docs/production/security-model.md

```diff
@@ -167,7 +167,7 @@ strength allowed is controlled by two settings in
   can rotate their own API key if it is accidentally compromised.
 
 - To properly remove a user's access to a Zulip team, it does not
-  suffice to change their password or deactivate their account in a
+  suffice to change their password or deactivate their account in an
   SSO system, since neither of those prevents authenticating with the
```

Comment:
It's possible "a Single Sign On system" would be a better fix.

Reply:
The correct expansion of SSO is “single sign-on”. The existence of an acronym doesn’t mean the expansion should be capitalized.